### PR TITLE
chore: update and re-enable sentry

### PR DESCRIPTION
**Pull request by wgwz**
## Description

Closes: regen-network/regen-registry#1540

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-registry/src/index.tsx [hunk 1]
```diff
@@ -42,6 +42,10 @@ const intercomId = process.env.REACT_APP_INTERCOM_APP_ID || '';
 // you can set this flag in local environments if testing changes to sentry.
 // currently we only want this flag set for the production environment.
 if (process.env.REACT_APP_SENTRY_ENABLED) {
+  const defaultSampleRate = '0.2';
```
Comment (wgwz) [inline web-registry/src/index.tsx:45]: this reduces the amount of "tracing" or in other words the amount of transactions sent from the app to sentry

Comment (wgwz) [reply web-registry/src/index.tsx:45]: here's some more info about the `BrowserTracing`:
https://docs.sentry.io/platforms/javascript/performance/instrumentation/automatic-instrumentation/

and the trace sampling rate:
https://docs.sentry.io/platforms/javascript/configuration/sampling/#setting-a-uniform-sample-rate

Comment (flagrede) [reply web-registry/src/index.tsx:45]: That's good to know.

**Review by flagrede — APPROVED** [review]
Looks good!

**Pull request merged**